recipes/harminv/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix="${PREFIX}" --enable-shared --with-blas=openblas
+make
+make install
```

Comment:
Can you run `make check`?

Reply:
`make check` runs the same test I was running in `meta.yaml`, so I changed `meta.yaml` to just do the existence tests, and now I'm running `make check`.